src/components/studies/session-creator/SessionCreator.tsx

```diff
@@ -136,8 +153,30 @@ const SessionCreator: FunctionComponent<SessionCreatorProps> = ({
     })
   }
 
-  const updateAssessments = (sessionId: string, assessments: Assessment[]) => {
+  const updateAssessments = async (
```

Comment:
so here, just have a (sessionId, prevAssessments, newAssessments) as params.
Then you just need to import/get resources for new assessments and then you can concat those arrays and update the assessments for session